airbyte-integrations/connectors/source-github/source_github/source.py

```diff
@@ -130,37 +130,39 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         authenticator = self._get_authenticator(config)
         repositories = self._generate_repositories(config=config, authenticator=authenticator)
         organizations = list({org.split("/")[0] for org in repositories})
-        full_refresh_args = {"authenticator": authenticator, "repositories": repositories}
-        incremental_args = {**full_refresh_args, "start_date": config["start_date"]}
+
         organization_args = {"authenticator": authenticator, "organizations": organizations}
-        default_branches, branches_to_pull = self._get_branches_data(config.get("branch", ""), full_refresh_args)
+        repository_args = {"authenticator": authenticator, "repositories": repositories}
+        repository_from_date_args = {**repository_args, "start_date": config["start_date"]}
```

Comment:
```suggestion
        repository_args_with_start_date = {**repository_args, "start_date": config["start_date"]}
```
IMO, it will be a little clearer this way